front/src/views/Schedule.vue

```diff
@@ -210,5 +207,9 @@ export default {
     .v-card {
         margin-bottom: 12px;
     }
+
+    .highlight {
+        background: #1f7087;
```

Comment:
there is no way to use a variable here ? :)

Reply:
Yes ! Its the same color as the datepiker card color, maybe we can use vuetify theme for that 🤔
https://vuetifyjs.com/fr-FR/customization/theme/

Reply:
https://github.com/ytvnr/gif-of-the-day/issues/109